src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -133,11 +134,13 @@ const SendFunds = ({
 
   let tokenSpendingLimit
   const handleSubmit = (values) => {
+    console.log(values)
     const submitValues = values
```

Comment:
Since you're touching this code, could you change this to `const submitValues = { ...values}`?